.vib/laravel/goss/vars.yaml

```diff
@@ -0,0 +1,13 @@
+binaries:
+  - php
+  - node
+directories:
+  - mode: "0775"
+    paths:
+      - /app
+  - paths:
+      - /opt/bitnami/php/bin
```

Comment:
There is no need to check this path given that you are already doing the binary check